templates/backup-swift.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Exits after first failure
+set -e
+
+timestamp=$(date +%Y%m%dT%H:%M:%S)
+source "{{ BACKUP_SWIFT_RC_DEST }}"
+
+# swift client bundled with ubuntu has no -D option, so we have to cd into target directory
+cd "{{ BACKUP_SWIFT_LOCAL_DIR }}"
+/usr/bin/swift download  "{{ BACKUP_SWIFT_CONTAINER }}"
```

Comment:
It would be nice to include container meta-data in the backup.  Simply sending the output of `swift stat BACKUP_SWIFT_CONTAINER` to a file will probably be enough.  We would have to restore that manually (since the output isn't in a machine-readable format), but at least we would have some record of the metadata.